modules/storages/lib/open_project/storages/engine.rb

```diff
@@ -203,7 +203,7 @@ def self.permissions
                              (prj_storage.project_folder_automatic? && !u.allowed_in_project?(:read_files, prj))
             next if hide_from_menu
 
-            icon = storage.provider_type_nextcloud? ? "nextcloud-circle" : "hosting"
+            icon = storage.provider_type_nextcloud? ? "nextcloud-circle" : "copy-documents"
```

Comment:
SO, we do replace the onedrive icon, too? not only the default files icon in the admin menu?

Reply:
@Kharonus I did. 
Thinking about it, maybe I should not. I will remove that for this PR and have it clarified first